### PR TITLE
Add openstack

### PR DESCRIPTION
**Pull request by neilmartindev**
## What does this PR do?
Add resource(s) 

### Description
Adding Openstack CERN guide and freecodecamp course
### Why is this valuable (or not)?
Insight into Openstack, how to use and operate it 
### How do we know it's really free?
Provided by CERN and freecodecamp free of charge.
### For book lists, is it a book? For course lists, is it a course? etc.
Course
## Checklist:
- [x] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [x] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [x] Include author(s) and platform where appropriate.
- [x] Put lists in alphabetical order, correct spacing.
- [x] Add needed indications (PDF, access notes, under construction).
- [x] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

File: courses/free-courses-en.md [hunk 1]
```diff
@@ -400,6 +401,12 @@
 * [IBM Cloud Essentials V3](https://cognitiveclass.ai/courses/ibm-cloud-essentials) - CognitiveClass.ai
 
 
+### Openstack
+
+* [Openstack CERN Admin guide](https://clouddocs.web.cern.ch/index.html) - CERN
+* [OpenStack Tutorial – Operate Your Own Private Cloud (Full Course)](https://www.youtube.com/watch?v=_gWfFEuert8) - FreeCodeCamp.org
```
Comment (eshellman) [inline courses/free-courses-en.md:407]: This would fit better in the books list

File: courses/free-courses-en.md [hunk 2]
```diff
@@ -400,6 +401,12 @@
 * [IBM Cloud Essentials V3](https://cognitiveclass.ai/courses/ibm-cloud-essentials) - CognitiveClass.ai
 
 
+### Openstack
+
+* [Openstack CERN Admin guide](https://clouddocs.web.cern.ch/index.html) - CERN
```
Comment (eshellman) [inline courses/free-courses-en.md:406]: It's usually not helpful to add a category for a single resource. Try to find another category, or see if there are other books in our lists that would fit into your category. I would just put this directly under cload computin and not create a separate category

**Review by eshellman — CHANGES_REQUESTED** [review]
For youtube, we prefer "playlist" links and the name of the producer/creator. docs/CONTRIBUTING.md#platforms-and-access-notes. the freecodecamp link is just under our 1-hour guideline for single video courses, but we would accept it.

File: courses/free-courses-en.md (hunk 2 above)
Comment (neilmartindev) [reply courses/free-courses-en.md]: np, moved the Openstack guide to books in the cloud computing category.

**Comment by eshellman** [comment]
Thanks, and welcome to free-programming-books!

**Pull request merged**